apps/users/templates/users/login_form.html

```diff
@@ -1,4 +1,6 @@
-<form method="post" action="{{ action }}" class="{% if is_ajax %}ajax-submit {% endif %}listing-footer">
+{% set login_source_form = switch_is_active('fxa-auth') and not form.errors %}
```

Comment:
I could check if the form was submitted in the view and pass this in. Would that be better?